src/modules/txbuilder/RadixTransactionBuilder.ts

```diff
@@ -303,15 +303,20 @@ export default class RadixTransactionBuilder {
      * @param  {string|number|Decimal} decimalQuantity
      */
     public mintTokens(
-        ownerAccount: RadixAccount, 
-        tokenReference: string | RRI, 
+        ownerAccount: RadixAccount,
+        tokenReference: string | RRI,
         decimalQuantity: string | number | Decimal) {
-        
+
         tokenReference = (tokenReference instanceof RRI)
             ? tokenReference
             : RRI.fromString(tokenReference)
 
         const tokenClass = ownerAccount.tokenDefinitionSystem.getTokenDefinition(tokenReference.getName())
+
+        if(!tokenClass) {
+            throw new Error(`ERROR: Token ${tokenReference.getName()} not found.`)
```

Comment:
I think this error could be a bit more precise - something like "Token definition XYZ not found in the owner account"

Specifically we need to be clear about the difference between tokens and token definitions(this is still an issue in many places, but at least going forward)

Reply:
Resolved